styles.css

```diff
@@ -8,24 +10,421 @@
-        etc.
+        4.3 Work
+        4.4 Services
+        4.5 Contact
+        4.6 Footer
     5. Media Queries
 ////////////////////////*/
 
 /* ------------------------------- 1. Global Styles -------------------------------- */
+:root {
+    --purple: #5722CD;
+    --light-purple: #F2F0FF; /* (voor de grote zwevende woorden, zoals "hello" en "services") */
+    --blue: #4845E4;
+    --pink: #ED4385;
+    --main-text-color: #0C0C0C;
+    --grey: #7A7A7A;
+    --light-grey: #F3F6F9;
+    --light-text: #FFF;
+}
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    font-family: 'Roboto', sans-serif;
+    font-size: 14px;
+}
+
+.outer-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+}
+
+.inner-container {
+    flex-basis: 1200px;
+    margin: 0 20px;
+    align-items: center;
+
+    display: flex;
+    flex-direction: column;
+}
+
 
 /* ------------------------------- 2. Typography ----------------------------------- */
+h1, h2, h3, h4, h5 {
+    font-family: 'Merriweather', serif;
+    font-weight: 400;
+}
+
+h1 {
+    font-size: 2.5rem;
+    margin-bottom: 0.8em;
+}
+h2 {
+    font-size: 2rem;
+    margin-bottom: 0.8em;
+}
+h3 {
+    font-size: 1.5rem;
+    margin-bottom: 0.8em;
+}
+
+p {
+    margin-bottom: 1em;
+}
+
+a {
+    text-decoration: none;
+    color: var(--purple);
+}
+
+a:hover {
+    color: var(--blue);
+}
 
 /* ------------------------------- 3. Layout --------------------------------------- */
 
 /* --------------- 3.1 Reusables ----------------------- */
+.light-text,
+.light-text a {
+    color: var(--light-text);
+}
+
+.purple-text {
+    color: var(--purple);
+}
+
+main section {
+    padding: 100px 0;
+}
+
+img {
+    width: 100%;
+}
+
+img.icon {
+    width: 25%;
+}
+
+main section:nth-of-type(2) {
+    background-color: var(--light-grey);
+    z-index: 1;
+    padding: 165px 0;
+}
+
+button {
+    background-color: var(--pink);
+    color: var(--light-text);
+    padding: 16px 25px;
+    border-radius: 25px;
+    border: none;
+    text-transform: uppercase;
+    font-size: 0.8rem;
+    box-shadow: 0 1px 12px rgba(0, 0, 0, 0.16);
+}
+
+button:hover {
+    box-shadow: 0 1px 12px rgba(0, 0, 0, 0.16) inset;
+    cursor: pointer;
+}
+.skew-bottom {
+    position: relative;
+}
+.skew-bottom::after {
+    background-color: #fff;
+    display: block;
+    height: 85px;
+    left: 0;
+    position: absolute;
+    right: 0;
+    bottom: -45px;
+    transform: skew(0deg, 2deg);
+    content: " ";
+    z-index: 2;
+}
 
 /* --------------- 3.2 Other --------------------------- */
 
 /* ------------------------------- 4. Areas ---------------------------------------- */
 
 /* --------------- 4.1 Header ------------------------- */
+.bg-highlight {
+    background: rgb(72, 69, 228);
+    background: url("assets/noise-texture.png"), linear-gradient(90deg, rgba(72, 69, 228, 1) 0%, rgba(87, 34, 205, 1) 100%);
+}
+
+.navigation {
+    margin: 20px;
```

Comment:
Ik zie dat je px en rem door elkaar gebruikt. Op zich niets mis mee, maar het is fijn om door de hele codebase heen (waar mogelijk) consistent te zijn qua eenheid